.github/workflows/basesystem.yml

```diff
@@ -1,62 +1,123 @@
 name: Build basesystem
 # Builds the Storm basesystem Docker images and deploys them to Dockerhub
+# Build is distributed across multiple runners based on:
+# https://docs.docker.com/build/ci/github-actions/multi-platform/#distribute-build-across-multiple-runners
 
 on:
   # needed to trigger the workflow manually
   workflow_dispatch:
 
+env:
+  IMAGE: movesrwth/storm-basesystem
+
 jobs:
-  buildDistro:
-    name: Build for ${{ matrix.distro.name }}
+  build:
+    name: Build ${{ matrix.image.name }} on ${{ matrix.platform }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        distro:
-          - {name: "ubuntu-20.04", baseImg: "ubuntu:20.04"}
-          - {name: "debian-11", baseImg: "debian:11"}
-          - {name: "ubuntu-22.04", baseImg: "ubuntu:22.04"}
-          - {name: "debian-12", baseImg: "debian:12"}
-          - {name: "ubuntu-24.04", baseImg: "ubuntu:24.04"}
-          - {name: "latest", baseImg: "ubuntu:rolling"}
+        image:
+          - {name: "ubuntu-20.04", baseImg: "ubuntu:20.04", file: "storm-basesystem/Dockerfile"}
```

Comment:
Do you still want the old distros in here>

Reply:
I want to keep the old distros for now such that we can still build ARM images for the older systems as well. After building these images once I will remove them from the workflow.